services/service-helpers/validator.js

```diff
@@ -23,6 +23,73 @@ v.validate = function(schema, config) {
   });
 };
 
+v.oneOf = function(options) {
```

Comment:
Let's make this `oneOfType`, do distinguish from `oneOf` which compares against definite values. 

And what do you think of using `arguments` instead of an array argument? Curious so I can be consistent with the decision.